### ISSUE TITLE
Example missing for configuration of Synology Photos

### ISSUE THREAD

**Issue by OlafKocanda** [home-assistant/home-assistant.github.io#27263]
### Feedback

I have tried to follow the description in [https://www.home-assistant.io/integrations/synology_dsm/#media-source](media source) to get Synology Photos as a media source.

I have entered the following in my configuration.yaml to see all images.

```
media_source:
  media-source://synology_dsm/2150XXXXXXXXXX/0/
```
Where 2150XXXXX is my unique ID I took from the media source folder:
![grafik](https://user-images.githubusercontent.com/58265203/236165922-95639deb-34dd-49c8-b8b0-92405ab3f181.png)

However nothing shows up, after configuring it.

![grafik](https://user-images.githubusercontent.com/58265203/236166041-69681cc5-87c1-4950-9c9e-94acb6acc1cd.png)

Can you please give an example of the configuration?

Thanks! 

### URL

https://www.home-assistant.io/integrations/synology_dsm/

### Version

2023.5.0

**Comment by mib1185** [comment]
the media browser is automatically registered in HA (_you do not need to configure something manually, just have the Synology Photo App installed on the NAS_) and media should be visible.

**Comment by OlafKocanda** [comment]
It is visible in the media, but I thought I need to select which album is shown in HA. The latter does not work.
Synology Photos is here:
![grafik](https://user-images.githubusercontent.com/58265203/236402479-e055a05e-37e4-4702-bc38-0f34a99b74bf.png)

The all images folder is also there:
![grafik](https://user-images.githubusercontent.com/58265203/236402567-c27d5931-7bc4-4c90-a984-9b3f1841c9de.png)

But the folder is empty:
![grafik](https://user-images.githubusercontent.com/58265203/236402617-2551034b-6c9d-4e0e-bb2b-d0aee63f97de.png)

That is why I thought that I did something wrong with my configuration?

```
media_source:
  media-source://synology_dsm/2150XXXXXXXXXX/0/
```

Where 2150XXXXX is my unique ID I took from the media source folder. Therefore an example would be really helpful!
